scripts/presubmit.sh

```diff
@@ -97,27 +97,31 @@ main() {
     go build ${goflags} ./...
 
     echo 'running go test'
+    # Install test deps so that individual test runs below can reuse them.
+    echo 'getting test deps'
+    go get -t
```

Comment:
Need "./..." for `go get -t` as well as `go test -i`.

Reply:
I think the go get ./... is actually already done in .travis.yml, so I've removed it.